py/kubeflow/kubeflow/ci/kfctl_e2e_workflow.py

```diff
@@ -532,7 +554,7 @@ def build(self):
            "--app_path=" + self.app_dir,
          ]
 
-    dependences = [build_kfctl["name"]]
+    dependences = [kf_second_apply["name"]]
```

Comment:
I think we want to run kubeflow-is-ready before we run a 2nd apply.
We want to make sure the first kfctl apply doesn't leave in KF in a broken state that is then fixed by the 2nd apply